packages/beacon-node/src/chain/validation/dataColumnSidecar.ts

```diff
@@ -247,7 +247,7 @@ export async function validateDataColumnsSidecars(
 
 /**
  * SPEC FUNCTION
- * https://github.com/ethereum/consensus-specs/blob/dev/specs/fulu/p2p-interface.md#verify_data_column_sidecar
+ * https://github.com/ethereum/consensus-specs/blob/ad36024441cf910d428d03f87f331fbbd2b3e5f1/specs/fulu/p2p-interface.md#L82
```

Comment:
there are a lot more links that reference `dev`

Reply:
yeah that bad enough, could at least update references you find when searching for `dev/specs/fulu`

Reply:
would prefer to do this under separate PR.  made an issue for it
https://github.com/ChainSafe/lodestar/issues/8171